packages/babel-helper-builder-react-jsx/src/index.ts

```diff
@@ -79,7 +79,7 @@ You can set \`throwIfNamespace: false\` to bypass this warning.`,
           "Fragment tags are only supported in React 16 and up.",
         );
       }
-      const callExpr = buildFragmentCall(path, state.file);
+      const callExpr = buildFragmentCall(path, state);
```

Comment:
This is the key fix suggested in https://github.com/babel/babel/issues/14884#issue-1351943248, other source changes are refactors so that the variable names are consistent with the typings.

Reply:
One small question: `state.file.get` also exists, why does it throw `pass.get(...) is not a function`?